app/views/posts/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>show user posts details id</h1>
\ No newline at end of file
+<section>
+  <%= render partial: 'shared/user_show', locals: {user: @user} %>
```

Comment:
- [ ] The partial is not rendered correctly, please verify that the trays of the design are shown as the wireframe shows.